klvdata/misb0601.py

```diff
@@ -68,9 +73,14 @@ class PrecisionTimeStamp(DateTimeElementParser):
     Precision Timestamp represented in the number of microseconds elapsed
     since midnight (00:00:00), January 1, 1970 not including leap seconds.
 
-    See MISB ST 0603 for additional details.
+    See MISB ST 0601.11 for additional details.

```

Comment:
The MISB standards are mostly active documents with revisions. Unless a specifc revision is critical, citing just the document number implies that the reader is intended to pull from the data management system (i.e. source) the latest revision. Also, if a revision is stated but isn't critical, the cost of maintaining this code/text increases as it will need to change everytime a revision to the cited document occurs. 

To summarize, unless critical, please cite standards using document numbers excluding revision identifiers.

Reply:
Ah okey!you know better those standards